src/CakeAdapter.php

```diff
@@ -344,15 +344,15 @@ public function fetchAll($sql)
     }
 
     /**
-     * Inserts data into the table
+     * Inserts data into a table.
      *
      * @param Table $table where to insert data
-     * @param array $columns column names
-     * @param $data
+     * @param array $row
+     * @return void
```

Comment:
That is the common guideline about using void. It means you cannot expect it to return anything, thus you must not return anything, the only return statement allowed is `return;` itself for "early returns".